components/observations/ObservationsFilterForm.tsx

```diff
@@ -372,11 +372,11 @@ export const ObservationsFilterForm: React.FunctionComponent<ObservationsFilterF
 
 export const matchesZone = (mapLayer: MapLayer, lat: number | null | undefined, long: number | null | undefined): string => {
   if (!lat || !long) {
-    return 'Not in any Forecast Zone';
+    return 'Unknown Zone';
```

Comment:
The original text was very log and wrapped
  awkwardly

when used in a title bar :D